packages/server-wallet/e2e-testing/scripts/start-ganache.ts

```diff
@@ -1,21 +1,31 @@
 import yargs from 'yargs/yargs';
 import {hideBin} from 'yargs/helpers';
 import {TEST_ACCOUNTS} from '@statechannels/devtools';
-import {writeFile} from 'jsonfile';
+import jsonfile, {writeFile} from 'jsonfile';
```

Comment:
nit: Can we make the imports consistent?

```suggestion
import jsonfile from 'jsonfile';
```

Reply:
ad3ec9a